classes/class-wc-rest-connect-stripe-account-controller.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Stripe_Account_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Stripe_Account_Controller extends WC_REST_Connect_Base_Controller {
+	protected $rest_base = 'connect/stripe/account';
+	private $stripe;
+
+	public function __construct( WC_Connect_Stripe $stripe, WC_Connect_API_Client $api_client, WC_Connect_Service_Settings_Store $settings_store, WC_Connect_Logger $logger ) {
+		parent::__construct( $api_client, $settings_store, $logger );
+		$this->stripe = $stripe;
+	}
+
+	public function get( $request ) {
+		$payload = $this->stripe->get_settings();
```

Comment:
What's an example use of this endpoint? Doesn't `WC_Connect_Stripe::get_settings()` require a URL parameter?

Reply:
True. This was never used in my test UI because I was fetching everything I needed in the call for the oauth URL. 

This GET method should return the default email and country values that the UI could use for auto-filling the fields. Since we're not using those at the moment, I'll just delete this GET method for now.